sdk/src/Core/Amazon.Util/AWSSDKUtils.cs

```diff
@@ -1316,12 +1363,7 @@ public static void Sleep(TimeSpan ts)
         /// </summary>
         /// <param name="value">Bytes to convert.</param>
         /// <returns>Hexadecimal string representing the byte array.</returns>
-        public static string BytesToHexString(byte[] value)
-        {
-            string hex = BitConverter.ToString(value);
-            hex = hex.Replace("-", string.Empty);
-            return hex;
-        }
+        public static string BytesToHexString(byte[] value) => ToHex(value, false);
```

Comment:
Not sure why we had both `ToHex` and `BytesToHexString`. Going with old codebase. The `BytesToHexString` is only called in [AmazonS3ResponseHandler.cs](https://github.com/aws/aws-sdk-net/blob/22a1ec96f18a31b72c4fe1bc79f8a37f2690c7a1/sdk/src/Services/S3/Custom/Internal/AmazonS3ResponseHandler.cs#L213). Since this is v4 I think we can just get rid of this method and update AmazonS3ReponseHandler.